libs/core/langchain_core/prompt_values.py

```diff
@@ -130,6 +131,22 @@ def to_messages(self) -> list[BaseMessage]:
         return [HumanMessage(content=[cast("dict", self.image_url)])]
 
 
+class DataPromptValue(PromptValue):
```

Comment:
There are 3 different names:

content_block
DataContentBlock
DataPromptValue

Any naming changes that could help reduce name variations would be great

Why not MultimodalPromptValue?